lib/prometheus_exporter/instrumentation/unicorn.rb

```diff
@@ -51,13 +51,13 @@ def collect_unicorn_stats(metric)
 
     def worker_process_count
       return nil unless File.exist?(@pid_file)
-      pid = File.read(@pid_file)
+      pid = File.read(@pid_file).to_i
 
-      return nil unless pid && pid.to_i > 0
+      return nil unless pid && pid > 0
```

Comment:
minor but I guess we want this now that we can not get nil. 

```suggestion
      return nil if pid < 1
```